plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveWriterFactory.java

```diff
@@ -735,7 +735,7 @@ public static String getFileExtension(HiveCompressionCodec compression, StorageF
         return compression.getHiveCompressionKind()
                 .filter(_ -> format.getOutputFormat().equals(HIVE_IGNORE_KEY_OUTPUT_FORMAT_CLASS))
                 .map(CompressionKind::getFileExtension)
-                .orElse("");
+                .orElse(format.getFileExtension());
```

Comment:
does this seem right ?

Reply:
My guess is you will want `name.formatExtension.compressionExtension` so you get `foo.txt.gx`

Reply:
that makes sense to me. updated